crates/open_ai/src/open_ai.rs

```diff
@@ -55,6 +59,12 @@ pub enum Model {
     #[serde(rename = "gpt-4o", alias = "gpt-4o-2024-05-13")]
     #[default]
     FourOmni,
+    #[serde(rename = "custom")]
+    Custom {
+        name: String,
+        #[serde(default = "custom_default_max_tokens")]
```

Comment:
I think we should make this a required field instead of giving it a default. We don't really have any basis for guessing what the token limit might be for an unknown model! 😄